src/App/AppContainer.js

```diff
@@ -92,13 +92,13 @@ class AppContainer extends Component {
     });
   };
 
-  render () {
+  render() {
     const { isFetchingResource } = this.props;
 
     if (!this.state.isValidPage) return <NotFoundPage />;
 
     return (
-      <div className="container-fluid">
+      <div>
         <div className="viewport">
           <div className="viewport-header">
             <Header toggleSavedResourcesPane={this.toggleSavedResourcesPane} />
```

Comment:
Somewhere here, the functionality of the spinner indicating loading is lost. I am not sure why, nothing you added should have changed it, but now the spinner doesn't show while loading the page. 99% sure its something in here.

Reply:
reworked the loading